src/main/java/com/synopsys/integration/detect/lifecycle/run/RunManager.java

```diff
@@ -172,6 +190,21 @@ public RunResult run(ProductRunData productRunData) throws DetectUserFriendlyExc
         return runResult;
     }
 
+    private Set<DetectTool> fetchEnabledTools() throws IntegrationException {
+        Set<DetectTool> enabledTools = new HashSet<>();
+        if (blackDuckServicesFactory != null) {
+            // Get features (tools) from blackduck
+            BlackDuckService blackDuckService = blackDuckServicesFactory.getBlackDuckService();
+            List<RegistrationFeaturesView> features = blackDuckService.getResponse(ApiDiscovery.REGISTRATION_LINK_RESPONSE).getFeatures();
```

Comment:
I believe we'd have to use /api/registration-summary, not /api/registration. Both seem to return the subset of data that we actually use. /api/registration-summary requires no user roles, where-as I think /api/registration requires access we can't assume the user will have. (There is no support for /api/registration-summary in blackduck-common etc. yet.)

Reply:
Related ticket: IDETECT-2208